src/pkg/packager/publish.go

```diff
@@ -62,68 +61,34 @@ func parseZarfLayerMediaType(filename string) string {
 // Authentication is handled via the Docker config file created w/ `zarf tools registry login`
 func (p *Packager) Publish() error {
 	p.cfg.DeployOpts.PackagePath = p.cfg.PublishOpts.PackagePath
-	if err := p.loadZarfPkg(); err != nil {
-		return fmt.Errorf("unable to load the package: %w", err)
-	}
 
-	paths := []string{
-		p.tmp.ZarfYaml,
-		filepath.Join(p.tmp.Images, "index.json"),
-		filepath.Join(p.tmp.Images, "oci-layout"),
-	}
-	// if checksums.txt file exists, include it
-	if !utils.InvalidPath(filepath.Join(p.tmp.Base, "checksums.txt")) {
-		paths = append(paths, filepath.Join(p.tmp.Base, "checksums.txt"))
-	}
-	// if p.tmp.SbomTar exists, include it
-	if !utils.InvalidPath(p.tmp.SbomTar) {
-		paths = append(paths, p.tmp.SbomTar)
```

Comment:
Without these checks we might have issues if someone tries to publish an older Zarf package.

Reply:
I don't think so, we had to do these checks before because we had to manually add all the layers that we wanted to publish because we were being inefficient with how we were unarchiving/archiving the layers within a package.

Because of the changes above, we are only unarchiving the package itself, meaning we can add all the things (that we found via the `filepath.Glob()` call) that was already inside of that tarball as layers that we are publishing.


@Noxsios Does that sound right to you too?

Reply:
Going to run this locally to verify all the correct layers are still included

Reply:
Doing this with a os.walk now to avoid this an issue.